egui/src/containers/collapsing_header.rs

```diff
@@ -19,6 +19,11 @@ pub(crate) struct State {
 
     /// Height of the region when open. Used for animations
     open_height: Option<f32>,
+
+    /// Whether collapsing controls and behavior is visible and
+    /// active. Set to `false` if you don't want to collapse your
+    /// windows.
+    pub enabled: bool,
```

Comment:
This is a weird place to store this boolean, considering it is not used in this file. In particular, if you just set up a `CollapsingHeader` and set `enabled = false` on it, it is still collapsible. I think it would make more sense to NOT have it here, but to keep `collapsible` to only `window.rs`.